tests/CMakeLists.txt

```diff
@@ -274,7 +274,7 @@ if(QUDA_MPI OR QUDA_QMP)
     get_test_ranks($ENV{QUDA_TEST_GRID_SIZE} QUDA_TEST_NUM_PROCS)
   endif()
   set(QUDA_CTEST_LAUNCH ${MPIEXEC_EXECUTABLE};${MPIEXEC_NUMPROC_FLAG};${QUDA_TEST_NUM_PROCS};${MPIEXEC_PREFLAGS}
-    CACHE STRING "CTest Launcher command for QUDA's tests")
+    CACHE STRING "CTest Launcher command for QUDA's tests" FORCE)
```

Comment:
This breaks CSCS_CI

Reply:
This was needed to avoid having to delete the cache file to change the grid size.  What’s a better way do this?

Reply:
For now, we decided the best thing to do is just to delete the entry in the cmake cache file.